framework/src/outputs/OutputWarehouse.C

```diff
@@ -153,6 +150,7 @@ OutputWarehouse::outputStep(ExecFlagType type)
 
   for (std::vector<Output *>::const_iterator it = _all_objects.begin(); it != _all_objects.end(); ++it)
     (*it)->outputStep(type);
+  flushConsoleBuffer();
```

Comment:
OK, I'm fine with this. So we have at least one hard flush every step. This is much better than std::endl sprinkled in places all over. Thanks